README.md

```diff
@@ -26,6 +26,9 @@
       --host=<host>    If starting server, specify ip address to bind to.
                        Default is 127.0.0.1
       --port=<port>    Specify server port number. Default is 9288
+      --remote-trigger Defer printing until page evaluates the following:
+                       window.cefPdf({request: "", onSuccess: function () {}, onFailure: function () {}});
```

Comment:
Hmm.. this is difficult as I'd rather it be simply `window.cefPdf()` but that doesn't work with the built in message router (it checks the types etc). I wonder if I can add another function onto `window` which simply calls the above.